mapie/utils.py

```diff
@@ -1373,3 +1373,19 @@ def check_n_samples(
              " int in the range [1, inf)"
              )
     return int(n_samples)
+
+
+def check_predict_params(
+    predict_params_used_in_fit: bool,
+    predict_params: dict,
+    cv: Optional[Union[int, str, BaseCrossValidator]] = None
+) -> None:
+    if (len(predict_params) > 0 and
+            predict_params_used_in_fit is False and
+            cv != "prefit"):
+        raise ValueError(
+            f"Using 'predict_param' '{predict_params}' "
+            f"without using one 'predict_param' in the fit method. "
+            f"Please ensure one 'predict_param' "
+            f"is used in the fit method before calling predict."
+        )
```

Comment:
Missing docstring

Reply:
Don't forget to use it in [mapie/regression/regression.py](https://github.com/scikit-learn-contrib/MAPIE/pull/471/files/3cdf6fe0a1679d1e182702ebe687a658e2003af5#diff-cc093002c5d5c9f31e9b108b77325c14147a5f9409299cbcbd8136de277fdcc3) line 625

Reply:
Indeed, thank you for your comment on the docstring